<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Регистрационный номер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Автор</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название книги</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год издания</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Издательство</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кол-во страниц</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataByFieldButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести данные о книгах
по полю</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Регистрационный номер</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Автор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Название</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дата</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Издатель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кол-во страниц</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор поля:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fullListButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести весь список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBookButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBookButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсортировать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
